frames/src/main/java/com/checkout/frames/component/cardholdername/CardHolderNameViewModel.kt

```diff
@@ -32,6 +32,10 @@ internal class CardHolderNameViewModel @Inject constructor(
     // Needed to prevent validation on focus switch for initial component state
     private var wasFocused = false
 
+    fun prepare() {
+        componentState.cardHolderName.value = paymentStateManager.cardHolderName.value
```

Comment:
shall we just leave this in `init`? unless we like to run `prepare` *not* right after vm is created.